cmd/migrtest/main.go

```diff
@@ -0,0 +1,94 @@
+package main
```

Comment:
I think it's quite clear.
It relies in a new repo as mentioned in PR description.